hack/make/.build-deb/rules

```diff
@@ -6,7 +6,7 @@ override_dh_gencontrol:
 	# if we're on Ubuntu, we need to Recommends: apparmor
 	echo 'apparmor:Recommends=$(shell dpkg-vendor --is Ubuntu && echo apparmor)' >> debian/docker-engine.substvars
 	# if we are building experimental we reccomend yubico-piv-tool
-	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~)")' >> debian/docker-engine.substvars
+	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~), libykcs11-1 (>= 1.1.0~)")' >> debian/docker-engine.substvars
```

Comment:
It's on their apt repo... Or ours now but I emailed them about it it's super weird


On Jan 7, 2016, 15:05 -0800, Tianon Gravi<notifications@github.com>, wrote:
>  
> Inhack/make/.build-deb/rules(https://github.com/docker/docker/pull/19168#discussion_r49139101):
>  
> >@@ -6,7 +6,7 @@ override_dh_gencontrol:># if we're on Ubuntu, we need to Recommends: apparmor>echo 'apparmor:Recommends=$(shell dpkg-vendor --is Ubuntu&&echo apparmor)'>>debian/docker-engine.substvars># if we are building experimental we reccomend yubico-piv-tool>- echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ]&&echo "yubico-piv-tool (>= 1.1.0~)")'>>debian/docker-engine.substvars>+ echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ]&&echo "yubico-piv-tool (>= 1.1.0~), libykcs11-1 (>= 1.1.0~)")'>>debian/docker-engine.substvars
>  
> I seehttps://packages.debian.org/sid/yubico-piv-tool, buthttps://packages.debian.org/sid/libykcs11-1is 404 -- where does this other package come from?
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/docker/docker/pull/19168/files#r49139101).
>  
>  
>